drivers/scheduler/anthos/anthos.go

```diff
@@ -847,28 +947,129 @@ func getExecPath() (string, error) {
 
 }
 
-func (anth *anthos) DeleteNode(node node.Node) error {
-	// TODO: Add implementation
-	return &errors.ErrNotSupported{
-		Type:      "Function",
-		Operation: "DeleteNode()",
+// getStoragePDBMinAvailableSet return storage min PDB value if set else ""
+func getStoragePDBMinAvailableSet() (int, error) {
+	pxOperator := operator.Instance()
+	schedOps, err := schedops.Get(SchedName)
+	if err != nil {
+		return 0, fmt.Errorf("failed to get driver for scheduler: %s. Err: %v", SchedName, err)
```

Comment:
Can return -1 here in case of error instead of 0. Same for all.

Reply:
Incorporated